cli/bench/space.go

```diff
@@ -54,3 +71,23 @@ func dropBenchmarkSpace(tarantoolConnection *tarantool.Connection) error {
 	}
 	return nil
 }
+
+// fillBenchmarkSpace fills benchmark space with a PreFillingCount number of records using connectionPool for fast filling
+func fillBenchmarkSpace(ctx context.BenchCtx, connectionPool []*tarantool.Connection) {
+	var waitGroup sync.WaitGroup
+	filledCount := 0
+	for i := 0; i < ctx.Connections; i++ {
+		waitGroup.Add(1)
+		go func(tarantoolConnection *tarantool.Connection) {
+			defer waitGroup.Done()
+			for filledCount < ctx.PreFillingCount {
```

Comment:
`filledCount` is used across multiple threads and must be handled under a mutex.

Reply:
Gorutins are not threads